src/Components/Landing/Landing.module.scss

```diff
@@ -0,0 +1,172 @@
+$title-color: rgba(0, 0, 0, 0.86);
+$button-background-color: #466eb6;
+$button-background-color_hover: #00CED1;
+
+@mixin flexDisplay($position, $direction) {
+  display: flex;
+  flex-direction: $direction;
+  align-items: center;
+  justify-content: $position;
+}
+
+//Main style file starts here
+
+.header {
+
+  img {
+    position: absolute;
+    left: 62.99%;
+    right: 7.2%;
+    top: 16rem;
+    bottom: 78.22%;
+
+    height: 43%;
+    width:  25%;
+  }
+
+  .title h1 {
+    position: absolute;
+    width: 30%;
+    height: 5%;
+    left: 9%;
+    top: 12rem;
+
+    /* Upperbound(desktop) */
+
+    font-family: "Poppins";
```

Comment:
Use the font-family from the variable present in `index.scss`

as 

```css
font-color: var(--gdsc-font-1);
```

Reply:
used the font